app/models/admin_configuration.rb

```diff
@@ -53,7 +53,7 @@ def self.get_ingest_docker_image_config
   # Ingest docker image getter
   def self.get_ingest_docker_image
     ingest_config = self.get_ingest_docker_image_config
-    if Rails.env != 'production' && ingest_config.present?
+    if !Rails.env.production? && ingest_config.present?
```

Comment:
I find the previous code more readable.

If you agree, could we disable this Rubocop rule?  If not, no worries, I'll adapt!

Reply:
This is a Rubocop suggestion, and we have mixed usage throughout the portal.  I'm updating them as we find them.  Personally I think it's easier to read, but I'm a Rubyist, so I get your objection.  I also prefer `unless` to a negative `if` as it reads more like language.